### PR TITLE
Add structured logging (first version)

### PR DESCRIPTION
**Pull request by Zelldon**
* Introduce new functions to log verbose or on info, use internally the old verbosity flags
  * allows simplifying certain code parts
* add zerolog for structured json logging (https://betterprogramming.pub/how-to-use-structured-json-logging-in-golang-applications-7fc5e2751dbd)
* add new flag to toggle structured logging for cli
  * useful when running worker

Future: Worker can add variables to the logger, like experiment name etc.

----

Example:

With json logging
```
$ ./zbchaos restart broker -v --jsonLogging
{"level":"debug","cli":"zbchaos","time":"2022-11-23T10:58:07+01:00","message":"Connecting to zell-chaos"}
{"level":"debug","cli":"zbchaos","time":"2022-11-23T10:58:07+01:00","message":"Running experiment in self-managed environment."}
{"level":"debug","cli":"zbchaos","time":"2022-11-23T10:58:07+01:00","message":"Successfully created port forwarding tunnel"}
{"level":"debug","cli":"zbchaos","time":"2022-11-23T10:58:07+01:00","message":"Found Broker zell-chaos-zeebe-1 as LEADER for partition 1."}
{"level":"debug","cli":"zbchaos","time":"2022-11-23T10:58:07+01:00","message":"Restarted zell-chaos-zeebe-1"}
```

Normal:
```
$ ./zbchaos restart broker -v
Connecting to zell-chaos
Running experiment in self-managed environment.
Successfully created port forwarding tunnel
Found Broker zell-chaos-zeebe-0 as LEADER for partition 1.
Restarted zell-chaos-zeebe-0
```

### CODE REVIEW

File: go-chaos/internal/logging.go [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package internal
+
+import (
+	"fmt"
+)
+
+func VerbosityLogging(text string, a ...any) {
```
Comment (oleschoenburg) [inline go-chaos/internal/logging.go:21]: :upside_down_face: names are too long IMO, I'd either move them to a new package called `log` so calls can look like `log.Verbose(...)` and `log.Info(...)`, or leave them in the `internal` package but rename the function to `LogInfo(...)` and `LogVerbose(...)` or similar.

**Review by oleschoenburg — APPROVED** [review]
Fantastic! :star_struck: 
Does that mean that logs are automatically parsed in google cloud log explorer or is there additional setup needed?

**Comment by Zelldon** [comment]
> Does that mean that logs are automatically parsed in google cloud log explorer or is there additional setup needed?

If it is deployed in gke it will automatically pushed to stackdriver, which will parse it afaik.

File: go-chaos/internal/logging.go (hunk 1 above)
Comment (Zelldon) [reply go-chaos/internal/logging.go:21]: Yeah was also not sure about them. I didn't wanted to conflict to much with the normal log package and functions

**Pull request merged**